content/docs/concepts/security/index.md

```diff
@@ -461,10 +460,24 @@ peers:
   - mtls: {}
 {{< /text >}}
 
-Currently, the mutual TLS setting doesn't require any parameters. Hence,
-`-mtls: {}`, `- mtls:` or `- mtls: null` declarations are treated the same. In
-the future, the mutual TLS setting may carry arguments to provide different
-mutual TLS implementations.
+The mutual TLS setting has an optional `mode` parameter that defines the
+strictness of the peer transport authentication:
+
+- `mode: STRICT` enforces that all peers must use mutual TLS. This declaration
+is identical to `-mtls: {}`, `- mtls:` or `- mtls: null`
+
+- `mode: PERMISSIVE` allows peers to use mutual TLS while still allowing
+peers to not use transport authentication. This mode is useful for transitioning
+existing workloads to peer authentication gradually, as it allows interoperation
+between mutual TLS and plaintext traffic.
+
+If this section is omitted completely, peers may not use transport
+authentication, and mutual TLS connections bound for the sidecar pod
+will be rejected. However clients are free to handle their own mutual
+TLS sessions at the application layer.
+
+In the future, the mutual TLS setting may carry additional arguments to
```

Comment:
To avoid confusing readers, please do not describe possible future behavior. Opt instead to update the docs when the new features are implemented.
```suggestion
```

Reply:
Noted, thanks. FYI this was already there, I merely moved it, it seemed strange to me as well.

Reply:
It was also wrong, given that we now do have an additional option: mode